core/src/components/split-pane/split-pane.ios.scss

```
@@ -11,6 +11,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side) {
+  @include rtl() {
+    border-right: 0;
+    border-left: var(--border);
+  }
+
```

```suggestion
  @include border(0, var(--border), 0, 0);
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-right: var(--border);` and `border-left: 0` lines below.